src/Gene/Composer.php

```diff
@@ -27,15 +27,15 @@ public function name(): string
     public function express(
         OperatingSystem $local,
         Server $target,
-        History $history
+        History $History
```

Comment:
Could you please modify the interface `Innmind\Genome\Gene` instead to use `$history` in order to have a coherent code style ?

Reply:
Fixed. I use the `$history` variable for the `Innmind\Genome\Gene` interface.

And all classes implemented above `Gene` interface will change into `$history` variable :).